packages/webcert/src/feature/careProvider/CareProviderModalContent.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { getUser } from '../../store/user/userSelectors'
+import { setUnit } from '../../store/user/userActions'
+import { SimpleTable } from '@frontend/common'
+import { ExpandableTableRow } from './ExpandableTableRow'
+
+export const CareProviderModalContent: React.FC = () => {
+  const dispatch = useDispatch()
+  const user = useSelector(getUser)
+
+  const handleChooseUnit = (event: React.MouseEvent) => {
+    const unitId = event.currentTarget.id
+
+    dispatch(setUnit(unitId))
+  }
+
+  return (
+    <>
+      {user?.careProviders.map((careProvider, idx) => {
+        const headings = [careProvider.name, 'Ej hanterade ärenden', 'Ej signerade utkast']
```

Comment:
Should these headings be generated from which pages the user can access? In that case the texts can be taken from the resource links

Reply:
According to the requirements the columns for "Ej hanterade ärenden" and "Ej signerade utkast" is not dependent on what the user can access. I will leave them hard coded for now and we can change it if the requirements change.